thunder/benchmarks/mistral.py

```diff
@@ -0,0 +1,176 @@
+import random
+import string
+import time
+
+from benchmark_scripts import Target
+
+from absl import logging, flags
+from absl.testing import absltest
+
+import torch
+from torch.optim import AdamW
+from torch.utils.data import DataLoader, IterableDataset
+
+from thunder.tests.litgpt_model import Config, GPT
+from transformers import AutoTokenizer
+
+flags.DEFINE_string("compile", default="eager", help="Specify compile option: thunder|inductor|eager")
```

Comment:
would there be an equivalent of argparse's [`choices`](https://docs.python.org/3/library/argparse.html#choices)?

Reply:
Yes it can be done with `lags.DEFINE_multi_string("multi_choice", ["a", "b", "c"] ...`.